public/gameSearch.php

```diff
@@ -60,14 +60,16 @@
 
         // echo " | ";
 
-        echo "<table><tbody>";
+        echo "<table class='table-highlight'><tbody>";
 
+        echo "<tr class='do-not-highlight'>";
         echo "<th>Rank</th>";
         echo "<th>Game</th>";
         echo "<th>Genre</th>";
         echo "<th>Publisher</th>";
         echo "<th>Developer</th>";
         echo "<th>Total Retro Ratio</th>";
+        echo "</tr>";
```

Comment:
In occurrences like these (the majority of them, actually), styling the first row as `.do-not-highlight` could be avoided if we just had the `th`s nicely wrapped in a `thead`. Sadly, `thead` pretty much isn't used anywhere in the legacy code...

Reply:
I'll put in the work and add the `thead` elements so we can remove the `.do-not-highlight` class in this PR if that's ok for you?

Reply:
Sadly, we can't just remove `.do-not-highlight` altogether due to the need of excluding other `tr` instances besides the topmost ones in tables (like comment fields).

That's why I opted for using it as an intermediate solution instead of toying with layout -- since we're going to move to blade templates and I imagine most of the "phony tables" will be transformed in generic containers instead.

Reply:
Sounds good, let's have it this way in the meantime 👍